ts/component/table-builder.ts

```diff
@@ -251,6 +263,7 @@ export class TableBuilder<ParentType, ChildType> {
     if (!this.definition.disablePageControls) {
       this.addSortControls(topControlsContainer);
       this.addFilterControls(topControlsContainer);
+      this.addDateFilterControls(topControlsContainer);
```

Comment:
These should not be separate controls. They should be handled the same as other filters for the table, except with a new `FilterType` to handle date input instead of text